internal/controller/controllers/hypershiftagentserviceconfig_controller.go

```diff
@@ -197,10 +231,10 @@ func (hr *HypershiftAgentServiceConfigReconciler) cleanStaleSpokeAgentInstallCRD
 	return nil
 }
 
-func (hr *HypershiftAgentServiceConfigReconciler) getAgentInstallCRDs(ctx context.Context, c client.Client) (*apiextensionsv1.CustomResourceDefinitionList, error) {
+func (hr *HypershiftAgentServiceConfigReconciler) getAgentInstallCRDs(ctx context.Context, c client.Client, namespace string) (*apiextensionsv1.CustomResourceDefinitionList, error) {
 	crds := &apiextensionsv1.CustomResourceDefinitionList{}
 	listOpts := []client.ListOption{
-		client.HasLabels{fmt.Sprintf("operators.coreos.com/assisted-service-operator.%s", hr.Namespace)},
+		client.HasLabels{fmt.Sprintf("operators.coreos.com/assisted-service-operator.%s", namespace)},
```

Comment:
it should be hr.namespace. i.e. the namespace in which the operator is installed on